packages/app/src/components/DBTracePanel.tsx

```diff
@@ -43,20 +42,41 @@ export default function DBTracePanel({
     },
   });
 
-  const { data: traceSourceData, isLoading: isTraceSourceLoading } = useSource({
-    id: watch('source'),
-  });
+  const { data: childSourceData, isLoading: isChildSourceDataLoading } =
+    useSource({
+      id: watch('source'),
+    });
 
   const { data: parentSourceData, isLoading: isParentSourceDataLoading } =
     useSource({
       id: parentSourceId,
     });
 
+  const logSourceData =
```

Comment:
Not 100% sure of the logic here but I'm pretty sure this is still causing a bug. If I'm searching from the trace table, and I open the Trace panel, logs do not show up.

We should be using `traceSourceId` and `logSourceId` to pull up the correlated source.

Reply:
It works fine in my local test. Can you tell me how do you reproduce this?
<img width="1252" alt="image" src="https://github.com/user-attachments/assets/6a03ea39-4c57-4f1c-8f93-03e4b0116b46" />

Reply:
No, I didn't. Selecting correlated log source only change `source` a.k.a child source, if you are in trace search page, your `parentSource` will be trace source.

<img width="1274" alt="image" src="https://github.com/user-attachments/assets/0fa293b1-468f-4d88-bbd4-948c2a85d9ec" />

Reply:
So when you're setting up your trace source, you want to set the correlated log source directly in the trace source, not ad-hoc in the trace panel, and you'll see this issue.
<img width="917" alt="image" src="https://github.com/user-attachments/assets/7f9e3be0-cb5d-40d9-859e-297e0aef3562" />

Reply:
oh I see, I also want to double check we are facing the same problem. My current issue is even I setup default Correlated Log Source, it does not select the correct one.
<img width="771" alt="image" src="https://github.com/user-attachments/assets/9d7babc6-1ea6-4970-9d08-b3e033734816" />


Here should be source `origin` instead of `trace2`
<img width="638" alt="image" src="https://github.com/user-attachments/assets/8bfb288f-f81a-430c-b5b5-c2ffa679d0f5" />

Reply:
@MikeShi42 just update the pr, this bug should be fixed now.